pkg/languages/defaults/python.go

```diff
@@ -0,0 +1,32 @@
+package defaults
+
+import (
+	"fmt"
+
+	"github.com/Azure/draft/pkg/reporeader"
+)
+
+type PythonExtractor struct {
+}
+
+// ReadDefaults reads the default values for the language from the repo files
+func (p PythonExtractor) ReadDefaults(r reporeader.RepoReader) (map[string]string, error) {
+	extractedValues := make(map[string]string)
+	files, err := r.FindFiles(".", []string{"*.py"}, 0)
+	if err != nil {
+		return nil, fmt.Errorf("error finding python files: %v", err)
+	}
+	if len(files) > 0 {
+		extractedValues["ENTRYPOINT"] = files[0]
```

Comment:
@davidgamero the only question I had is - ENTRYPOINT is the only defaults for python projects ? or this will be added or enhanced further ?

Reply:
ah this is just an example extractor, we can definitely expand this further! since the PR mainly adds the reporeader/extractor pattern I just wanted to have an example